server/odc-service/src/main/java/com/oceanbase/odc/service/sqlcheck/rule/MySQLAffectedRowsExceedLimit.java

```diff
@@ -120,14 +122,38 @@ public List<DialectType> getSupportsDialectTypes() {
         return Arrays.asList(DialectType.MYSQL, DialectType.OB_MYSQL);
     }
 
+    /**
+     * MySQL checks the count of value list. Process mode: case1: For INSERT INTO ... VALUES(...)
+     * statements, ODC checks the count of value list. case2: For INSERT INTO ... SELECT ... statements,
+     * ODC runs EXPLAIN statements to get affected rows.
+     *
+     * @param explainSql target sql
+     * @return affected rows
+     */
+    private long getMySqlAffectedRowsByCount(String explainSql, JdbcOperations jdbc) {
+
+        explainSql = explainSql.toLowerCase();
+
+        if (explainSql.contains("value")) {
```

Comment:
what if sql statement contains other 'value' , e.g. identifer & string value

Reply:
maybe we need to reconsider the matching strings here, e.g. only match `select`